src/extension.ts

```diff
@@ -418,6 +432,17 @@ function launchMetals(
         case "metals-model-refresh":
           compilationDoneEmitter.fire();
           break;
+        case "metals-open-window":
```

Comment:
just left a comment in metals that it might be better to rename it to `open-folder` / `open-directory` or something like that. If you rename it in metals it needs change here too. If not then ignore :)